src/features/pagination/js/pagination.js

```diff
@@ -368,8 +410,6 @@
           $scope.$on('$destroy', dataChangeDereg);
 
           var setShowing = function () {
```

Comment:
set showing is called in lines [415](https://github.com/chatcher/ui-grid/blob/29fdb7cd485607e0c7e579df82880b4aceae0d35/src/features/pagination/js/pagination.js#L415) and [432](https://github.com/chatcher/ui-grid/blob/29fdb7cd485607e0c7e579df82880b4aceae0d35/src/features/pagination/js/pagination.js#L432). One of those usages also creates a watch, I would argue that if this function can definitely stay empty that you should remove this function, the watch on line 415 and the usage on line 432. Also, the call to destroy said watch on line [438](https://github.com/chatcher/ui-grid/blob/29fdb7cd485607e0c7e579df82880b4aceae0d35/src/features/pagination/js/pagination.js#L438)